### DISCUSSION TITLE
Native support for numpy's date & time types

### DISCUSSION THREAD

**Discussion by erezinman** [pydantic/pydantic#3548] [Feature Request]
Hi, 
I do not know if that's a feature request or a bug, but numpy's native datetime types are not supported. For instance, if the model has a parameter that's annotated with `np.timedelta64`, the following error is raised
> RuntimeError: no validator found for <class 'numpy.timedelta64'>, see `arbitrary_types_allowed` in Config

These types are essentially 64-bit integers that aren't much different from other `pd.Timestamp` and `pd.Timedelta`. It could've been great if the these types were supported, and would have an automatic conversion between them and the usual `datetime.datetime` and `datetime.timedelta` types.

**Comment by TKaluza** [comment]
https://github.com/pydantic/pydantic/issues/6512

Right now it just get dropped.
